bootCampTopic3/bootCampTopic3Ex123/src/main/java/maven/exercise/bootCampTopic3Ex123/ShoppingCart.java

```diff
@@ -0,0 +1,52 @@
+package maven.exercise.bootCampTopic3Ex123;
+
+import java.util.ArrayList;
+
+
+
+public class ShoppingCart {
+	
+	private static ArrayList<Item> items = new ArrayList<Item>();	
+	private static ShoppingCart shoppingcart = null;
+
+	public ShoppingCart() {	
+	}
+	
+	public static ShoppingCart getInstance() {
+		if (shoppingcart == null) {
+			shoppingcart = new ShoppingCart();
+		}
+		return shoppingcart;
+	}
+	
+	
+	public ArrayList<Item> getItems() {
+		return items;
+	}
+
+	public void setItems(ArrayList<Item> items) {
+		ShoppingCart.items = items;
+	}
+	
+	public void addItem (Item i){
+		items.add(i);
+	}
+	
+	public void deleteItem (String name){
```

Comment:
In this method you can retur a boolean value, when the method is succefull or not